### ISSUE TITLE
Android error

### ISSUE THREAD

**Issue by yfuks** [taskrabbit/react-native-parsed-text#4]
Hello, i just try to use this module on an Android device and i get the folowing error :

![error](https://cloud.githubusercontent.com/assets/9282988/11688907/8c00fd24-9e8f-11e5-8304-fe4c8ddd6dfd.png)

there is my code (realy simple) : 

```javascript
'use strict';

var React = require('react-native');
var ParsedText = require('react-native-parsed-text');
var {
  Component,
  Text,
  View
} = React;

class Home extends Component {
  constructor (props) {
    super(props);
  }
  render() {
    return (
      <View style={{flex: 1}}>
        <Text style={{marginVertical: 20, marginHorizontal: 10}}>Welcome</Text>
        <ParsedText style={{color: 'black', fontSize: 15}} parse={  [ {type: 'url', style: {color: '#456'}, onPress: () => {}} ] } >hello</ParsedText>
      </View>
    );
  }
}

module.exports = Home;
```

My api version : API 18

So is this module supposed to work on Android or did i miss something ?

**Comment by jrichardlai** [comment]
I thought this should work out of the box for Android, definitely planning to support Android. 
Thanks for letting me now.

**Comment by yfuks** [comment]
Nice !
Really look for it !

**Comment by yfuks** [comment]
Any idea on the timeline for this ?

**Comment by jrichardlai** [comment]
Sorry pretty busy at the moment, which version of react native are you using ?

**Comment by yfuks** [comment]
0.15.0 actually

**Comment by jrichardlai** [comment]
Okay fixed it should be good for `0.0.8` version
